static/js/actions/index_page.js

```diff
@@ -0,0 +1,5 @@
+import { sendGoogleAnalyticsEvent } from '../util/util';
+
+export function event() {
```

Comment:
Would suggest renaming this to something that is a bit more descriptive like `emitPageview` or similar.